languages/go/exercises/concept/nil/nil_test.go

```diff
@@ -0,0 +1,139 @@
+package nil
+
+import "testing"
+
+func TestEmptyInterface(t *testing.T) {
+	test := struct {
+		name string
+		want interface{}
+	}{
+		"EmptyInterface",
+		nil,
+	}
+	t.Run(test.name, func(t *testing.T) {
+		if got := EmptyInterface(); got != test.want {
+			t.Errorf("EmptyInterface() = %v, want %v", got, test.want)
+		}
+	})
+}
+
+func TestEmptyMap(t *testing.T) {
+	t.Skip()
```

Comment:
One more thing: on the Go track we don't use test skipping. We always give the student all the feedback.